src/components/swapv2/styleds.tsx

```diff
@@ -219,7 +219,7 @@ export function SwapCallbackError({ error }: { error: string }) {
     <SwapCallbackErrorInner>
       <Alert style={{ marginBottom: 'auto' }} />
       <AutoColumn style={{ flexBasis: '100%', margin: '10px 0 auto 8px' }}>
-        <Text fontSize="16px" fontWeight="500" color={theme.red}>
+        <Text fontSize="16px" fontWeight="500" color={theme.red} style={{ lineHeight: '24px' }}>
```

Comment:
Text component also has lineHeight props, we dont need inline style here

Reply:
agree